hr_announcement/models/res_users.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2023 Tecnativa - Pilar Vargas
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, models
+
+
+class ResUsers(models.Model):
+    _inherit = "res.users"
+
+    @api.model
+    def announcement_user_count(self):
+        # res = super().announcement_user_count()
+        employee_announcements = self.env["announcement"].search_read(
+            [
+                ("announcement_type", "=", "employee"),
+                ("in_date", "=", True),
+                ("id", "not in", self.env.user.read_announcement_ids.ids),
```

Comment:
But you can write to the related employee user the announcement as it's done with the specific_user type so you don't have to go down to this logic: https://github.com/OCA/server-ux/blob/8311de64e311120a758db38cabf8993bb86907cf/announcement/models/announcement.py#L63-L71